src/pages/index.tsx

```diff
@@ -49,7 +49,7 @@ export default function Home(): JSX.Element {
     >
       <HomepageHeader />
       <main>
-        <Hackathon />
+        <!-- <Hackathon /> -->
```

Comment:
```suggestion
        {/* <Hackathon /> */}
```
should be commented out like this, otherwise fails to build